language-support/ts/codegen/tests/ts/build-and-lint-test/src/__tests__/test.ts

```diff
@@ -601,6 +601,63 @@ test("create + fetch & exercise", async () => {
   expect(nonTopLevelContracts).toEqual([nonTopLevelContract]);
 });
 
+test("exercise using explicit disclosure", async () => {
+  const aliceLedger = new Ledger({
+    token: ALICE_TOKEN,
+    httpBaseUrl: httpBaseUrl(),
+  });
+  const bobLedger = new Ledger({
+    token: BOB_TOKEN,
+    httpBaseUrl: httpBaseUrl(),
+  });
+
+  const payload: buildAndLint.Main.ReferenceData = {
+    p: ALICE_PARTY,
+  };
+  const contract = await aliceLedger.create(
+    buildAndLint.Main.ReferenceData,
+    payload,
+  );
+  // The JSON API does (not) yet expose contract metadata so we read it directly through the gRPC API.
```

Comment:
Should we put in a TODO and/or ticket number to switch this over once JSON API supports it?

Reply:
Do you have an issue that I can reference here?

Reply:
I think this would be the relevant ticket
https://digitalasset.atlassian.net/browse/LT-5

Reply:
thanks added